types/pouchdb-core/index.d.ts

```diff
@@ -55,6 +55,15 @@ declare namespace PouchDB {
             rev: RevisionId;
             status: Availability;
         }
+        interface RevisionDiffList {
+            [documentId : string]: string[];
+        }
+        interface RevisionDiff {
+            [change : string]: string[];
```

Comment:
Thanks for reviewing @geppy.

Looking at the PouchDB source, it just fires an xhr off to CouchDB and returns whatever the response is via the promise/callback. The [CouchDB docs](http://docs.couchdb.org/en/2.0.0/api/database/misc.html#db-revs-diff) document the response key as either "missing" or "possible_ancestors".

That aside, I've tried putting an enum together, however my IDE is flagging the code with an error: "TS1023: An index signature parameter type must be string or number". 

```
enum RevisionDiffStatus {'missing', 'possible_ancestors'}

interface RevisionDiff {
    [Change : RevisionDiffStatus]: string[];
}
```

Looking this error up, it doesn't look like Typescript supports enumerations as index types.

I'm somewhat new to TypeScript however and may be doing something wrong. Could you advise how it should be accomplished?